tests/saitests/sai_qos_tests.py

```diff
@@ -1714,13 +1714,24 @@ def runTest(self):
         tos |= ecn
         ttl = 64
         default_packet_length = 64
-        pkt = simple_tcp_packet(pktlen=default_packet_length,
-                                eth_dst=router_mac if router_mac != '' else dst_port_mac,
+        pkt_dst_mac = router_mac if router_mac != '' else dst_port_mac
+        pkt = simple_ip_packet(pktlen=default_packet_length,
+                                eth_dst=pkt_dst_mac,
                                 eth_src=src_port_mac,
                                 ip_src=src_port_ip,
                                 ip_dst=dst_port_ip,
                                 ip_tos=tos,
                                 ip_ttl=ttl)
+
+        print >> sys.stderr, "dst_port_id: %d, src_port_id: %d" % (dst_port_id, src_port_id)
+        # in case dst_port_id is part of LAG, find out the actual dst port
+        # for given IP parameters
+        dst_port_id = get_rx_port(
```

Comment:
@smaheshm, have you verified if the LAG will still stay up during this test on Th? This test will run for a longer duration on Th since we need lot more packets to fill up the shared pool

Reply:
Didn't try on TH. Will check it out and update.

Reply:
@neethajohn This code doesn't change behavior for any existing test case, affects only multi ASIC which I've already verified.

Reply:
Looks fine.

```
samaddik@str-serv-acs-14:/var/sonic-mgmt/tests$ pytest --count=3 qos/test_qos_sai.py  --testbed=vms3-t1-dx010-1 --inventory=../ansible/str --testbed_file=../ansible/testbed.csv --host-pattern=str-dx010-acs-1 --module-path=../ansible/library --disable_loganalyzer --skip_sanity -k  PgSharedWatermark
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
========================================================================================================================================================================== test session starts ===========================================================================================================================================================================
platform linux2 -- Python 2.7.17, pytest-4.6.5, py-1.10.0, pluggy-0.13.1
ansible: 2.8.12
rootdir: /var/sonic-mgmt/tests, inifile: pytest.ini
plugins: forked-1.3.0, metadata-1.11.0, xdist-1.28.0, html-1.22.1, repeat-0.9.1, ansible-2.2.2
collected 57 items / 51 deselected / 6 selected                                                                                                                                                                                                                                                                                                                          

qos/test_qos_sai.py ......                                                                                                                                                                                                                                                                                                                                           [100%]

============================================================================================================================================================================= warnings summary =============================================================================================================================================================================
/usr/local/lib/python2.7/dist-packages/_pytest/config/__init__.py:538
  /usr/local/lib/python2.7/dist-packages/_pytest/config/__init__.py:538: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: tests.common.dualtor
    self.import_plugin(import_spec)
    
-- Docs: https://docs.pytest.org/en/latest/warnings.html
========================================================================================================================================================== 6 passed, 51 deselected, 1 warnings in 1098.24 seconds ==========================================================================================================================================================
```

Reply:
Thank you for verifying